dotcom-rendering/src/components/Discussion/CommentForm.tsx

```diff
@@ -140,6 +140,27 @@ const commentAddOns = css`
 	list-style-type: none;
 `;
 
+const boldComment = css`
+	font-weight: bold;
+`;
+
+const italicComment = css`
+	font-style: italic;
+`;
+
+const strikethroughComment = css`
+	text-decoration: line-through;
+`;
+
+const codeComment = css`
+	/* stylelint-disable-next-line property-disallowed-list -- we don't want this font from Source */
+	font-family: monospace;
+`;
+
+const linkComment = css`
+	text-decoration: underline;
+`;
```

Comment:
Actually, for a single property change like this it would be [more performant to use the `style` prop](https://emotion.sh/docs/best-practices#use-the-style-prop-for-dynamic-styles).

Reply:
that's interesting, thanks! I think the relevant part is [this](https://emotion.sh/docs/best-practices#use-the-style-prop-for-dynamic-styles):

> By dynamic styles, we mean styles that change frequently or are **unique to a single element.**

(emphasis mine).